msisdn-gateway/e2e-test/fabfile.py

```diff
@@ -1,10 +1,12 @@
-from fabric.api import *
-import fabric
+from fabric.api import *  # noqa
+# import fabric
```

Comment:
Yeah, I tried fixing the `import *` but I guess I wasn't adding enough imports because it wasn't working for me. We can fix it in a future PR or try and dive in now. Wasn't sure if the msisdn stuff is final, or still WIP stuff.